### PR TITLE
SSH-Panel Package

### PR DESCRIPTION
**Pull request by Haiquan-27**
add SSH-Panel

- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [ ] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [ ] If my package is a syntax it doesn't also add a color scheme. ***
- [ ] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My package Package functions:

1. View, Add, Edit files on remote SSH host
2. View remote file detailed properties
3. Browse with HTML panel like sidebar
4. Verify remote host fingerprint
5. Support password and public-private key pair authentication methods

The function is similar to the *RemoteTree*

difference:
* support on the Linux and Windows
* support all available openssh server operating systems
* the details of the server can be displayed

### CODE REVIEW

**Comment by Haiquan-27** [comment]
Should I wait for the automatic test?

File: repository/s.json [hunk 1]
```diff
@@ -3523,6 +3523,21 @@
 				}
 			]
 		},
+		{
+			"name": "SSH-Panel",
+			"details": "https://github.com/Haiquan-27/SSH-Panel",
+			"author": "Haiquan",
+			"issues": "https://github.com/Haiquan-27/SSH-Panel/issues",
+			"readme": "https://raw.githubusercontent.com/Haiquan-27/SSH-Panel/main/README.md",
```
Comment (braver) [inline repository/s.json:3531]: You don't need "issues" or "readme" here since they're just the defaults for GitHub.

**Comment by braver** [comment]
- Can your dependency be handled through the dependencies feature of package control, or maybe by the user installing it on their machine?
- Would be nice to be able to access the settings though the main menu.

**Comment by Haiquan-27** [comment]
I don't support Mac because I haven't found a way to compile library files on Mac. In fact, as long as I can compile library files, I can support Mac platform. I haven't solved this problem recently, and it may be solved in the next version.

Can I accept my current submission?

**Comment by braver** [comment]
Sure, sounds reasonable. Please have a look at the other feedback too.

File: repository/s.json (hunk 1 above)
Comment (Haiquan-27) [reply repository/s.json]: i have deleted " issues" and "readme"

**Comment by braver** [comment]
> Can your dependency (on paramiko) be handled through the dependencies feature of package control, or maybe by the user installing it on their machine?

🤔❔

**Comment by Haiquan-27** [comment]
Dependencies can be accessed by users from [here](https://github.com/Haiquan-27/SSH-Panel-doc-annex) Self download, which provides the dependent files required by the platform supported by the package

In addition, because Windows requires additional python DLL file, so dependencies feature of package control is not applicable

**Pull request merged**